crates/symbolicator/src/cache.rs

```diff
@@ -131,11 +131,49 @@ pub struct GcsSharedCacheConfig {
     pub source_key: Arc<GcsSourceKey>,
 }
 
-/// A remote cache that can be shared between symbolicator instances.
+/// The backend to use for the shared cache.
```

Comment:
nit: maybe this should be in its own module

Reply:
hmm, this crossed my mind as well but it's still config objects for the cache.  and this would be pretty small.  i don't think there's a need just yet.